projects/public-search/src/app/patron-profile/patron-profile-loans/patron-profile-loans.component.html

```diff
@@ -18,38 +18,68 @@
 <section id="loans-section" class="container my-2">
   <div class="d-none d-lg-block">
     <div class="row bg-dark rounded text-light p-2">
-      <div class="col-4 pl-0" translate>Title</div>
-      <div class="col-4" translate>Library</div>
-      <div class="col-2" translate>Due date</div>
-      <div class="col-2">&nbsp;</div>
+      <div class="col-4 pl-0">
+        <ng-core-sort-list
+          [options]="[
+          {value:'title', label:'Title' | translate, icon:'fa-sort-alpha-asc'},
+          {value:'-title', label:'Title (desc)' | translate, icon:'fa-sort-alpha-desc'}
+          ]"
+          [useIcon]="true"
+          (selectChange)="selectingSortCriteria($event)">
+        </ng-core-sort-list>
+      </div>
+      <div class="col-4">
+        <ng-core-sort-list
+          [options]="[
+          {value:'library', label:'Library' | translate, icon:'fa-sort-alpha-asc'},
+          {value:'-library', label:'Library (desc)' | translate, icon:'fa-sort-alpha-desc'}
+          ]"
+          [useIcon]="true"
+          (selectChange)="selectingSortCriteria($event)">
+        </ng-core-sort-list >
+      </div>
+      <div class="col-3">
+        <ng-core-sort-list
+          [options]="[
+          {value:'duedate', label:'Due date' | translate, icon:'fa-sort-numeric-asc'},
+          {value:'-duedate', label:'Due date (desc)' | translate, icon:'fa-sort-numeric-desc'}
+          ]"
+          [useIcon]="true"
+          (selectChange)="selectingSortCriteria($event)">
+        </ng-core-sort-list>
+      </div>
+      <div class="col-1">&nbsp;</div>
     </div>
   </div>
 
+  <table class="table">
+  <tbody>
+    <ng-container *ngIf="!loaded; else display">
+      <div class="row">
+        <div class="col mt-2 font-weight-bold" translate>Loading in progress</div>
+      </div>
+    </ng-container>
```

Comment:
Seems strange to include a `div` as direct child of `tbody`.
`table` (+rows+colums) should be only included if loading is finished as before.